doc/sphinx/language/core/assumptions.rst

```diff
@@ -202,10 +202,12 @@ has type :n:`@type`.
    :name: ‘ident’ already exists. (Axiom)
    :undocumented:
 
-.. warn:: @ident is declared as a local axiom
+.. warn:: Use of "Variable" or "Hypothesis" outside sections behaves as "#[local] Parameter" or "#[local] Axiom".
 
    Warning generated when using :cmd:`Variable` or its equivalent
    instead of :n:`Local Parameter` or its equivalent.
+   This warning errors by default, but it can be convenient to turn it
+   off during debuging activities.
```

Comment:
```suggestion
   This message is an error by default, but it can be convenient to turn it
   off during debugging activities.
```

IIUC it can be an error, a warning or completely hidden.  If it can no longer be a warning, please tweak the wording here and above to reflect that.